frontend/tests/fixtures.ts

```diff
@@ -61,4 +69,46 @@ export const test = base.extend<Fixtures>({
     await deleteUser(context.request, tempUser.id);
     await context.close();
   },
+  tempProject: async ({ page }, use, testInfo) => {
+    const titleForCode =
+      testInfo.title
+      .replaceAll(' ','-')
+      .replaceAll(/[^a-z-]/g,'');
+    const code = `test-${titleForCode}-${testInfo.testId}`;
+    const name = `Temporary project for ${testInfo.title} unit test`;
+    const loginData = {
+      emailOrUsername: 'admin',
+      password: testEnv.defaultPassword,
+      preHashedPassword: false,
+    }
+    const response = await page.request.post(`${testEnv.serverBaseUrl}/api/login`, {data: loginData});
+    expect(response.ok()).toBeTruthy();
+    const gqlResponse = await executeGql(page.request, `
+      mutation {
+        createProject(input: {
+          name: "${name}",
+          type: FL_EX,
+          code: "${code}",
+          description: "temporary project created during the ${testInfo.title} unit test",
+          retentionPolicy: DEV
+        }) {
+          createProjectResponse {
+            id
+            result
+          }
+          errors {
+            __typename
+            ... on DbError {
+              code
+              message
+            }
+          }
+        }
+      }
+`);
+    const id = (gqlResponse as {data: {createProject: {createProjectResponse: {id: UUID}}}}).data.createProject.createProjectResponse.id;
```

Comment:
At this point I was just trying to get Typescript to stop complaining about code I knew was working. :laughing: But yeah, a generic (and named) type passed to executeGql would make that look nicer.

Reply:
Since you resolved the conversation, I'm assuming this is out of scope for this PR. So I created https://github.com/sillsdev/languageforge-lexbox/issues/570 to track this suggestion so we wouldn't lose track of it.